src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENTAL_COST = 40;
+  const SHORT_DISCOUNT_PERIOD = 3;
+  const LONG_DISCOUNT_PERIOD = 7;
+  const SHORT_DISCOUNT = 20;
+  const LONG_DISCOUNT = 50;
```

Comment:
Move CONSTANTS outside the function.
CONSTANT - This is a variable that is created only 1 time and never changes.
In ur case, it is created every time the function is called

Reply:
Thank you, fixed